pkgs/development/python-modules/pick-colour-picker/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv
+, fetchFromGitHub
+, buildPythonPackage
+, pygobject3
+, pycairo
+, glib
+, gtk3
+, gobject-introspection
+, wrapGAppsHook
+}:
+
+buildPythonPackage rec {
+  pname = "pick-colour-picker";
+  version = "1.5.0-3ec940";
+
+  src = fetchFromGitHub {
+    owner = "stuartlangridge";
+    repo = "ColourPicker";
+    rev = "3ec9406d787ce373f6db0d520ed38a921edb9473";
+    sha256 = "04l8ch9297nhkgcmyhsbg0il424c8vy0isns1c7aypn0zp0dc4zd";
+  };
+
+  propagatedBuildInputs = [
+    glib
+    pygobject3
+    pycairo
+    gtk3
+    gobject-introspection
```

Comment:
not sure about propagating some of these, means that you may introduce binaries to the PATH which are not meant to be there.

Reply:
Seems like moving these causes things...

with gobject-introspection

![screenshot](https://user-images.githubusercontent.com/7735145/70847497-69931b00-1e65-11ea-842d-9726df98bb2a.png)

when it's not propagated, just nativeBuildInputs

![screenshot](https://user-images.githubusercontent.com/7735145/70847506-7d3e8180-1e65-11ea-842e-4f34608fb4bd.png)

Kinda feel like touching these is always going to blow up

Reply:
Fixed that with `strictDeps = false`

Reply:
@FRidh you probably would know this

Reply:
See https://github.com/NixOS/nixpkgs/issues/56943

Reply:
Is this resolved, then?